cirq/aqt/aqt_sampler.py

```diff
@@ -0,0 +1,226 @@
+import json
+import time
+import uuid
+from typing import Iterable, List, Union
+
+import numpy as np
+from requests import put  # TODO: Specify as a dependency
+from cirq import XPowGate, YPowGate, XXPowGate
```

Comment:
Import `ops` instead of values from `ops`. This reduces the chance of cyclic dependency.